modules/ROOT/pages/mongodb/mongodb-connector-6-0-design-center.adoc

```diff
@@ -0,0 +1,78 @@
+= MongoDB Design Center Configuration - Mule 4
```

Comment:
Instructions for Edit:

We have a major connector update for the MongoDB Connector going out on Wednesday. There are four topics ready for a standard review in this PR: https://github.com/mulesoft/docs-connectors/pull/628/
mongodb-connector-6-0.doc
mongodb-connector-6-0-design-center.doc
mongodb-connector-6-0-studio.doc
mongodb-connector-6-0-xml-maven.doc
If possible, please have these ready sometime on Tuesday. There are three more topics coming, which I will get to you as soon as possible (one is very small).